CorsixTH/Src/th_gfx_sdl.h

```diff
@@ -527,7 +529,9 @@ class sprite_sheet {
       @param iFlags Flags to apply for drawing.
   */
   void draw_sprite(render_target* pCanvas, size_t iSprite, int iX, int iY,
-                   uint32_t iFlags);
+                   uint32_t iFlags,
+                   size_t game_ticks = 0u,
```

Comment:
I'm not a fan of the default value here, however draw_sprite seems to have several call sites not from animation_manager which would complicate knowing the value of game_ticks. It is only used when you also have a non-none effect so maybe that's okay.